config/routes.rb

```diff
@@ -31,6 +31,13 @@
         get '/random' => 'random#show'
       end
 
+      namespace :invoice_items do
```

Comment:
Before submission, let's order these namespaces and resources similarly (just added waffle card)